### DISCUSSION TITLE
en/post/astro-sitemap/

### DISCUSSION THREAD

**Discussion by giscus** [bepyan/bepyan.me.v2#43] [General]
# en/post/astro-sitemap/

A Sitemap assists search engines in easily crawling a website by designating its pages. Astro provides a handy plugin `@astrojs/sitemap` to generate your sit...

https://bepyan.me/en/post/astro-sitemap/

**Comment by f33w** [comment]
Heya, good read. Waiting for update for Astro 5 syntax
